actix-http/src/body/mod.rs

```diff
@@ -87,6 +87,7 @@ mod tests {
     }
 
     impl ResponseBody<Body> {
+        #[allow(dead_code)]
```

Comment:
Wouldn't it be better to leave a comment on why is it kept? Otherwise, it has chances to either be removed earlier than needed or not removed when it's time.